test/unit/core/resource/Repository/ClassRepositoryTest.php

```diff
@@ -147,7 +147,7 @@ private function createClass(string $uri = null, string $label = null): core_ker
         $class
             ->expects($uri !== null ? $this->once() : $this->never())
             ->method('getUri')
-            ->willReturn($uri);
+            ->willReturn((string)$uri);
```

Comment:
Why we need typecast for URI?

Reply:
It's not nullable in the original method.